src/options/settings/components/KeyboardShortcuts.js

```diff
@@ -0,0 +1,135 @@
+import * as React from 'react'
+import { Checkbox } from '../../../common-ui/components'
+import * as utils from 'src/content-tooltip/utils'
+const styles = require('./settings.css')
+
+class KeyboardShortcuts extends React.PureComponent {
+    state = {
+        shortcutsEnabled: true,
+        highlightShortcut: 'h',
+        linkShortcut: 'l',
+        toggleSidebarShortcut: 'r',
+        toggleHighlightsShortcut: 'h',
+        createAnnotationShortcut: 'a',
+        highlightShortcutEnabled: true,
+        linkShortcutEnabled: true,
+        toggleSidebarShortcutEnabled: true,
+        toggleHighlightsShortcutEnabled: true,
+        createAnnotationShortcutEnabled: true,
+    }
+    async componentDidMount() {
+        const keyboardShortcutsState = await utils.getKeyboardShortcutsState()
+        this.setState(keyboardShortcutsState)
+    }
+    handleCheckboxToggle = async e => {
+        const name = e.target.name
+        const value = e.target.checked
+        this.setState({ [name]: value })
+        await utils.setKeyboardShortcutsState({
+            ...this.state,
+            [name]: value,
+        })
+    }
+
+    handleInputChange = async e => {
+        e.persist()
```

Comment:
What's the need for this line? Could it be used in `handleCheckboxToggle`? If so, `handleCheckboxToggle` and `handleInputChange` contain the exact same code and can probably be unified (but maybe there's a good reason why that's not a good idea)